modules/default/weather/current.njk

```diff
@@ -54,16 +54,32 @@
             </div>
         {% endif %}
     </div>
-    {% if (config.showFeelsLike or config.showPrecipitationAmount) and not config.onlyTemp %}
+    {% if (config.showFeelsLike or config.showPrecipitation or config.showPrecipitationAmount or config.showPrecipitationProbability) and not config.onlyTemp %}
```

Comment:
not happy with these if constructs...

can we get rid of the the old "config.showPrecipitationAmount" here by setting the other values correctly in weatherjs when starting up? something similar like we did for useKmh / useBeaufort https://github.com/MichMich/MagicMirror/blob/684c727430586d88a48202f9276f63ed0ae3decc/modules/default/weather/weather.js#L79

Reply:
I agree, but I'm not sure removing `config.showPrecipitationAmount`. Personally I consider using `config.showPrecipitationAmount` or `config.showPrecipitationProbability`. I find the `config.showPrecipitation` to take too much horizontal space. Have a look at the examples in the description. Maybe we don't need to offer displaying both values with `config.showPrecipitation`?

On the other hand, the probability can be hidden with CSS (`.precipitationProbability { display: none; }`). Maybe this is enough? If it is we can consider removing `config.showPrecipitationProbability` as well? By adding another `<span>` around the parenthesis it can be hidden together with the amount, resulting in a clean display of the probability.

Reply:
I mixed something up, didnt realize that `showPrecipitationAmount` was the old config option. 
So my reasoning now is: 
since precipation isnt really use dmuch (otherwise more users would have complained :-) I 'd just add the `showPrecipitationProbability` config option and let the user decide if they want to show one, both or none (default). no deprecation warning necessary then and the code is a little smaller.

Reply:
Agreed :blush: